openfermioncirq/gates/four_qubit_gates.py

```diff
@@ -12,12 +12,60 @@
 
 """Gates that target four qubits."""
 
-from typing import Optional, Union
+
+from typing import Optional, Union, Tuple
 
 import numpy
 
 import cirq
 
+from cirq import ops
```

Comment:
I'd guess/expect that has to do with the possibility of circular imports within Cirq, but circular dependency isn't possible in this case.

Reply:
@idk3 is correct. In code outside of Cirq source, `import cirq` is preferred.

Reply:
Okay, I changed the imports to the top level. Out of curiosity, why are circular imports not a concern here?

Reply:
We can't have a circular dependency because Cirq never imports from OpenFermion-Cirq. In Cirq, submodules might use parts of other submodules, which creates a risk of circular imports.

Reply:
Right, duh. Thanks for explaining.